stylesheets/_mixin.focus.scss

```diff
@@ -53,8 +53,11 @@
 
 @mixin pulsar-range-focused {
     border: 0;
-    box-shadow: none;
-    outline: none;
+    border-radius: 0;
```

Comment:
Perhaps it would make sense to remove this higher up, seeing as it's an inert condition for range sliders?

Reply:
Needed to override default browser focus styling in a couple of browsers.